packages/boot/test/fixtures/service-class.artifact.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export class GreetingService {
+  greet(whom: string = 'world') {
+    return Promise.resolve(`Hello ${whom}`);
+  }
+}
```

Comment:
Are we going to bind plain service class during boot? I don't see the logic.

Reply:
Not yet. The logic here is to ensure that the booter is picking only Providers and excludes regular classes, there is an integration test for that.

https://github.com/strongloop/loopback-next/blob/9098ad3ade0b226e90291c0f3365b290bd508a66/packages/boot/test/integration/service.booter.integration.ts#L23-L26

Suggestions are welcome on how to make this more obvious to people reading this code.

Maybe it's a sign that I should enhance `ServiceMixin` and the booter to handle both `app.serviceProvider(GeocoderServiceProvider)` and `app.service(GreeterService)` cases, even though `app.service` may not be used by any real LB4 app? IDK.

Reply:
For now, let's just add a comment to `GreeterService` saying it will be excluded from discovery as we only allow providers so far.